datahub-web-react/src/app/ingest/source/IngestionSourceList.tsx

```diff
@@ -204,22 +204,22 @@ export const IngestionSourceList = () => {
                 });
         } else {
             // Create
-            const newSource = {
-                urn: PLACEHOLDER_URN,
-                name: input.name,
-                type: input.type,
-                config: null,
-                schedule: {
-                    interval: input.schedule?.interval || null,
-                    timezone: input.schedule?.timezone || null,
-                },
-                platform: null,
-                executions: null,
-            };
-            addToListIngestionSourcesCache(client, newSource, pageSize, query);
             createIngestionSource({ variables: { input } })
                 .then((result) => {
                     message.loading({ content: 'Loading...', duration: 2 });
+                    const newSource = {
+                        urn: result?.data?.createIngestionSource || PLACEHOLDER_URN,
+                        name: input.name,
+                        type: input.type,
+                        config: null,
```

Comment:
(if we could have the full config in there this would be better. my hunch is that if you click the entry and the config is null, the recipe etc won't work properly. here's how you can test this assumption: 

1. Set the timeout for the refetch super high locally. 
2. Create new source.
3. Click 'Edit'